src/chrome/content/rules/info.gov.hk.xml

```diff
@@ -0,0 +1,21 @@
+<!--
+	For other HK government coverage, see GovHK.xml.
+
+	Nonfunctional hosts in *.info.gov.hk:
+		- gia.info.gov.hk (m)
+		- www-censtatd.3dns.info.gov.hk (m)
+		- www-legis.3dns.info.gov.hk (t)
+		- www-youth.3dns.info.gov.hk (m)
```

Comment:
See also `^info.gov.hk`, `www.info.gov.hk` and `webcast.info.gov.hk`

Reply:
It has been a while since last update...
Please have a look on the latest commit, many thanks!

Reply:
Thanks!